ray-operator/config/samples/ray_v1alpha1_rayservice-alb-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Rename the file to `rayservice-alb-ingress.yaml`

Reply:
The convention is to name the YAML file as `{CR type}.{feature}.yaml`. For example, `ray-service.autoscaler.yaml` indicates that the YAML file contains a RayService CR with autoscaling enabled. The `ray_v1alpha1_rayservice.yaml` does not adhere to the naming convention in KubeRay.